### PR TITLE
refactor: styles

### PR DESCRIPTION
**Pull request by spider0061**
## Description

refactor and divide _helper.scss into small files.

## Related issues and discussion
#60 

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by punwai** [comment]
Please make sure that the commit message and PR message are all lower case.

**Comment by punwai** [comment]
You can run git rebase -i HEAD~1 to rename your commit message.

**Comment by spider0061** [comment]
> Please make sure that the commit message and PR message are all lower case.

Done

**Comment by spider0061** [comment]
All files have passed linting the build error is due to time out.

File: src/assets/styles/_loader.scss [hunk 1]
```diff
@@ -1,46 +1,45 @@
 .loader-wrapper {
```
Comment (abhaychawla) [inline src/assets/styles/_loader.scss:1]: This file is correct and does not require any changes. We can have nested scss classes. Please remove the changes from this file.

File: src/assets/styles/_helper.scss [hunk 2]
```diff
@@ -1,1726 +1,17 @@
...
-  color: rgba(0, 0, 0, 0.87);
-  line-height: 1.5;
-  font-family: Roboto, Helvetica Neue, sans-serif;
-
-  table {
-    border: 1px solid rgba($color: #000000, $alpha: 0.12);
-    width: 100%;
-    tr:nth-child(even) {
-      background-color: #f2f2f2;
-    }
```
Comment (abhaychawla) [inline src/assets/styles/_helper.scss:35]: I am not able to locate this in any file. Can you please check for all such differences in the files and ensure nothing is missing? Also, make sure if anything was nested it remains nested.

**Comment by spider0061** [comment]
> Have a look at the review. Also, make sure you use only `imperative` tense in your commit message. Have a look at the commit message guidelines and update the commit message description (body) accordingly.

@abhaychawla I have made the changes. Please have a look.

File: src/assets/styles/_display.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  top: 0;
+  bottom: 0;
+}
+
+.lr-0 {
+  left: 0;
+  right: 0;
+}
+
+.fullwidth {
```
Comment (abhaychawla) [inline src/assets/styles/_display.scss:77]: We have a separate scss file for width and height which makes this redundant (you can combine them or keep both and remove redundancies)

File: src/assets/styles/_align.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
+html,
+body {
+  position: relative;
+  overflow: hidden;
+}
```
[the remark this answers is not in the document]
Comment (abhaychawla) [reply src/assets/styles/_align.scss]: Overflow is still here, but you have a separate file for that.

File: src/assets/styles/_width-height.scss [hunk 5]
```diff
@@ -1,4 +1,50 @@
...
+
+.full-height { 
+  height: 100%; 
+}
+
+html,
+body {
+  width: 100%;
+  height: 100%;
+  -webkit-tap-highlight-color: transparent;
```
Comment (abhaychawla) [inline src/assets/styles/_width-height.scss:21]: Move this to some other file, as it's not related to width or height.

Comment (spider0061) [reply src/assets/styles/_width-height.scss]: @abhaychawla I have moved this to miscellaneous file.

**Comment by spider0061** [comment]
@abhaychawla done

**Pull request merged**